src/views/api-docs-view/components/path-truncation-aside/index.tsx

```diff
@@ -6,6 +6,17 @@
 import DevDotContent from 'components/dev-dot-content'
 import CodeBlock from '@hashicorp/react-code-block'
 
+/**
+ * Truncates HCP Vault Secrets API operation paths for clarity.
+ * Intended to be used with a `<PathTruncationAside />` in each operation intro.
+ */
+function truncateVaultSecretsOperationPath(path: string) {
+	return path.replace(
```

Comment:
⛏️ Could add add a bit more clarity as to what the regex should match on in the comment?

Reply:
🙌 Great call! Added this in 👍